src/utils/hibp.js

```diff
@@ -147,8 +156,48 @@ async function loadBreachesIntoApp (app) {
   log.info('done-loading-breaches', 'great success 👍')
 }
 
+async function downloadBreachIcons (breaches) {
+  const breachDomains = breaches
+    .map(breach => breach.Domain)
+    .filter(breachDomain => breachDomain.length > 0)
+  const logoFolder = pathResolve(dirname(fileURLToPath(import.meta.url)), '../client/images/logo_cache/')
```

Comment:
Use `../../dist` instead of `../client` here. See Slack thread

Reply:
OK, I've added a `LIVE_RELOAD` environment variable that is set to `'true'` when we're running with `npm run dev`, and then determine which directory to use based on that: 6e6e2ec9